pallets/messages/src/mock.rs

```diff
@@ -26,7 +28,8 @@ type Block = frame_system::mocking::MockBlock<Test>;
 
 pub const INVALID_SCHEMA_ID: SchemaId = 65534;
 pub const IPFS_SCHEMA_ID: SchemaId = 50;
-
+// following number is not random it is the account_id of INVALID_CALLER_NAME in benchmarking.rs
+pub const INVALID_CALLER_ACCOUNT: u64 = 15032055221142339925;
```

Comment:
not sure how substrate maps string account names to account ids, but it seems like there is a 1-1 relationship between them. Found this by logging what was coming back